packages/happychat-connection/src/use-happychat-auth.ts

```diff
@@ -3,7 +3,7 @@ import { useQuery } from 'react-query';
 import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
 import type { HappychatAuth } from './types';
 
-export const happychatAuthQueryKey = 'getHappychatAuth-' + Date.now();
+export const happychatAuthQueryKey = 'getHappychatAuth';
```

Comment:
You want to export the full key array here. This export is then used as a key in `queryClient.fetchQuery`, and React Query v4 accepts only array keys. It would have to be fixed later anyway, during the v3 -> v4 upgrade.

Reply:
cc @flootr in case that lands as-is and needs to be changed to an array.

Reply:
Fixed 👍🏼